prooftree.typ

```diff
@@ -10,9 +10,10 @@
     side: left,
     padding: 0.2em,
   ),
+  line-stroke: 0.5pt,
   ..rules
 ) = {
-  // Check the types of `spacing` and `label`.
+  // Check the types of the parameters.
```

Comment:
Okay, maybe it's not exactly true that we're checking every parameter, but only the "first level" ones

Reply:
I just wanted to avoid someone looking at these asserts and wondering wtf is going on, so this is fine.